patterns/template-archive-vertical-header-blog.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * Title: Archive for the right aligned blog
```

Comment:
I think all the right-aligned template names could use some adjusting.
Is it right-aligned, right aligned, or, is it the vertical header that should be in focus?

Reply:
I'll take a stab.